src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplTest.CaseDTO.json

```diff
@@ -0,0 +1,103 @@
+[
```

Comment:
Why the extra data file? Is this used by anything?

Reply:
no I don't need it after all - now deleted